.github/workflows/integration.yml

```diff
@@ -1,11 +1,8 @@
 name: integration-test
 
 on:
-  # Trigger the workflow on push or pull request,
+  # Trigger the workflow on pull request,
   # but only for the main branch
-  push:
-    branches:
-      - main
```

Comment:
this we wanted on every push to main

Reply:
do we need to run the integration test again when its pushed to main? i think running the integration test on pull request to main is enough? so we don't need to run it twice.

Reply:
but what about direct commits to main? (although we should discourage that but still possible)

Reply:
since there's still possibility, i'm reverting this back